subprojects/core/src/integTest/groovy/org/gradle/api/FinalizerTaskIntegrationTest.groovy

```diff
@@ -214,13 +214,13 @@ class FinalizerTaskIntegrationTest extends AbstractIntegrationSpec {
         expect:
         2.times {
             fails 'a'
-            failure.assertHasDescription """Circular dependency between the following tasks:
-:a
-\\--- :c
-     \\--- :b
-          \\--- :a (*)
-
-(*) - details omitted (listed previously)"""
+            failure.assertHasDescription """|Circular dependency between the following tasks:
+                                            |:a
+                                            |\\--- :c
+                                            |     \\--- :b
+                                            |          \\--- :a (*)
+                                            |
+                                            |(*) - details omitted (listed previously)""".stripMargin()
```

Comment:
❔- I haven't tested if this will be compatible with JDK17. I've got some problems in promotion with conflicting Groovy/Java APIs.